pkg/chain/ethereum/connect.go

```diff
@@ -92,25 +95,24 @@ func connectWithClient(
 		)
 	}
 
-	ec := &ethereumChain{
-		config:           config,
+	c := &Chain{
```

Comment:
Two questions:
1. Why not keep it as `ethereumChain` and have `ethereumBeaconChain` and `ethereumEcdsaChain`? I think it's easier to navigate through implementation files when it's implicitly denoted this is Ethereum implementation.
2. Why do we export it?

Reply:
This change was made in the initial implementation approach I've taken for this PR to reuse common Ethereum chain implementation in the random beacon and ecdsa packages, along with keeping it functional for the legacy code.